be/src/exprs/array_functions.cpp

```diff
@@ -752,6 +752,58 @@ class ArrayHasImpl {
             return true;
         }
     }
+
+    template <bool NullableElement, bool NullableTarget, typename ElementColumn>
+    static uint8 __process_seq(const ElementColumn& elements, uint32 element_start, uint32 element_end,
+                               const ElementColumn& targets, uint32 target_start, uint32 target_end,
+                               const NullColumn::Container* null_map_elements,
+                               const NullColumn::Container* null_map_targets) {
+        using ValueType = std::conditional_t<std::is_same_v<ArrayColumn, ElementColumn> ||
+                                                     std::is_same_v<MapColumn, ElementColumn> ||
+                                                     std::is_same_v<StructColumn, ElementColumn>,
+                                             uint8_t, typename ElementColumn::ValueType>;
+        [[maybe_unused]] auto is_null = [](const NullColumn::Container* null_map, size_t idx) -> bool {
+            return (*null_map)[idx] != 0;
+        };
+        if (element_end < target_end) {
+            return false;
+        }
```

Comment:
this is not true, as your tests case just consider single array cases. Taking an array column as example, 
elem: [null][null] (its offsets are [0,0,0]
target: [1,2,3,4],[null] (its offsets are [0,4,4]

array_contains_seq(elem, target) should return false, true. but your answer sould be wrong to false, false.

Reply:
if(element_end-element_start > target_end-target_start)